Patch: 71_date_overflows.patch
Author: Lionel Elie Mamane <lionel@mamane.lu>
React sensibly on integer overflow in date handling (Closes: #326024, CVE-2005-4153)
Index: mailman-2.1.12/Mailman/Handlers/Scrubber.py
===================================================================
--- mailman-2.1.12.orig/Mailman/Handlers/Scrubber.py	2009-03-02 20:50:19.000000000 +0100
+++ mailman-2.1.12/Mailman/Handlers/Scrubber.py	2009-03-02 20:50:20.000000000 +0100
@@ -143,7 +143,7 @@
                      }.get(parts[3], 0)
             day = int(parts[4])
             year = int(parts[6])
-        except (IndexError, ValueError):
+        except (OverflowError, IndexError, ValueError):
             # Best we can do I think
             month = day = year = 0
         datedir = '%04d%02d%02d' % (year, month, day)
